deconstrst/deconstrst.py

```diff
@@ -29,6 +34,26 @@ def build(srcdir, destdir):
 
     return app.statuscode
 
+def get_conf_builder():
+    conf_file = open('conf.py')
+
+    try:
+        conf_data = conf_file.read()
+    finally:
+        conf_file.close()
```

Comment:
You can do this bit more cleanly with a [`with` statement](http://preshing.com/20110920/the-python-with-statement-by-example/):

```python
with open('conf.py') as conf_file:
    conf_data = conf_file.read()
```

It'll automatically close it for you. :tada: